frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -34,11 +36,40 @@ const TermPlanner = () => {
   const planner = useSelector((state: RootState) => state.planner);
   const degree = useSelector((state: RootState) => state.degree);
 
-  const [termsOffered, setTermsOffered] = useState<Term[]>([]);
-  const [isDragging, setIsDragging] = useState(false);
+  const [draggingCourse, setDraggingCourse] = useState<string | undefined>();
```

Comment:
can we set for the initial state to be an empty string or null rather than undefined?